waterlog/src/Pages/Home.js

```diff
@@ -6,12 +6,14 @@ import TankLevels from "./../Components/TankLevels";
 class HomeComponent extends Component {
   render() {
     return (
-      <div>
+      <div className="home">
         <MapComponent />
-        <Wastage />
-        <TankLevels />
+        <div className="wastage-tank-wrapper">
+          <Wastage />
+          <TankLevels/>
+        </div>
       </div>
     );
   }
 }
-export default HomeComponent;
+export default HomeComponent;
```

Comment:
empty line above this line and empty line EOF

Reply:
done